### PR TITLE
fix: ui bugs with asset messages sending [AR-933] [AR-1451]

### PR DESCRIPTION
**Pull request by gongracr**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-933" title="AR-933" target="_blank">StoryAR-933</a>  Sending files
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Design identified several UI issues when checking the whole image/asset sending flow. Namely:

1. Image placeholder:
   a. the preview image should have rounded border as specified [here](https://www.figma.com/file/UgKehvwIbiIiQ2CCEdsVuf/Android?node-id=2221%3A143658)
   b. portrait images are shown as landscape (rotated 90 degree)  -> Can't really reproduce it
2. File placeholder:
   a. The filetype and “Tap to download” texts should have gray-70 color (called “Secondary Text” in  the semantic color palette) as specified [here](https://www.figma.com/file/UgKehvwIbiIiQ2CCEdsVuf/Android?node-id=5277%3A95980)
   b. Filetype text position should be closer to the icon (4px)
3. After a file or image was selected by the user in the file browser to be sent in the chat, the message inputfield should be closed completely (as shown [here](https://www.figma.com/file/UgKehvwIbiIiQ2CCEdsVuf/Android?node-id=2221%3A143658))

#### How to Test
Just launch the app, choose a conversation and try to send some asset and image messages.

### Notes (Optional)

It seems the (not) scrolling issue is also present on text, so might need to refactor that too if agreed.

### Attachments (Optional)

Captura de Pantalla 2022-04-25 a las 17 38 56

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-535/1/) **failed**.

File: app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt [hunk 1]
```diff
@@ -212,7 +209,15 @@ private fun ConversationScreenContent(
         messageText = messageText,
         onMessageChanged = onMessageChanged,
         onSendButtonClicked = onSendButtonClicked,
-        onSendAttachment = onSendAttachment,
+        onSendAttachment = {
+            coroutineScope.launch {
+                onSendAttachment(it)
+                // We add some extra delay to allowing some time to insert the message in the database, display it in the screen and be able
+                // to scroll to the bottom of the chat with the new list of messages
+                delay(2000)
```
Comment (yamilmedina) [inline app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt:217]: Maybe we could extract this value to a constant or var, either of those, should do it =)

**Review by yamilmedina — COMMENTED** [review]
I think there's an issue with `ConversationsViewModelTest` but other than that and a small magic number, great work 👏 💯

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-535/2/) **failed**.

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-535/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3199360)(3199360)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3199360)(3199360)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt [hunk 2]
```diff
@@ -212,7 +209,15 @@ private fun ConversationScreenContent(
...
-        onSendAttachment = onSendAttachment,
+        onSendAttachment = {
+            coroutineScope.launch {
+                onSendAttachment(it)
+                // We add some extra delay to allowing some time to insert the message in the database, display it in the screen and be able
+                // to scroll to the bottom of the chat with the new list of messages
+                delay(MESSAGE_SCROLL_DELAY)
+                lazyListState.animateScrollToItem(0)
+            }
+        },
```
Comment (trOnk12) [inline app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt:220]: looks a bit sketchy to me :P is there no other way to do it ? what if updating the database takes longer ? I know it shoudlnt be possible, but who knows. 

Why can't we control the scroll from the viewmodel ? as like send the attachment, wait some time for asynchronous sending and then once it is done scroll to the bottom ?

**Review by trOnk12 — APPROVED** [review]
looks sexy 🔥

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-535/5/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3201377)(3201377)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3201377)(3201377)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [6](https://10.10.124.134/job/wire-android-reloaded/job/PR-535/6/) **failed**.

**Comment by AndroidBob** [comment]
Build [8](https://10.10.124.134/job/wire-android-reloaded/job/PR-535/8/) **failed**.

**Comment by AndroidBob** [comment]
Build [9](https://10.10.124.134/job/wire-android-reloaded/job/PR-535/9/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3201861)(3201861)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3201861)(3201861)-dev-release.apk)

**Pull request merged**